pkg/managed/clusterroles_test.go

```diff
@@ -0,0 +1,281 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+package managed
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+	"github.com/verrazzano/verrazzano-operator/pkg/monitoring"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	"github.com/verrazzano/verrazzano-operator/pkg/types"
+	"github.com/verrazzano/verrazzano-operator/pkg/util"
+	rbacv1 "k8s.io/api/rbac/v1"
+	v1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"reflect"
+	"testing"
+)
+
+// TestCleanupOrphanedClusterRoles tests the cleanup of orphaned cluster roles
```

Comment:
ok this is good, but it made me wonder what exactly this func does - is it removing clusterroles that it does not expect? i was wondering how it would know if any given role ought to be there or not.  it is perfectly valid for the to be a clusterrole with no clusterolebinding.  so i went and read the func. turns out what it is doing is removing clusterroles which are labeled with the name of a non-existent verrazzanoBinding.  feels like that func has a name that is not very descriptive, and this test i guess just suffers by association.

Reply:
I guess that the author of this function thought that "orphaned" was an appropriate term for for a cluster role that is labeled with the name of a non-existent verrazzano binding.  That seemed reasonable to me.  Are you saying that is not appropriate?  I'm really not sure what to do here.  Should I rename the function?  CleanupClusterRolesWithNonExistentVerrazzanoBinding?

Reply:
let's do nothing, we can catch it in the next go-around